### PR TITLE
New improved data and removing data when AL/HI weren't states.

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
I created a new item in my personal space. We can just move it over to the viz space later.

### CODE REVIEW

**Review by jordansread — COMMENTED** [review]
Is it public? Do you not have write access to the vizlab community?

**Comment by dblodgett-usgs** [comment]
Yep. and nope. The old one is also public...

**Comment by jordansread** [comment]
OK, we'll get you access. Was asking about public because otherwise we can't build.

**Comment by dblodgett-usgs** [comment]
👍

File: viz.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ fetch:
     id: histWaterUseAWUDS-data
     location: cache/awuds.zip
     fetcher: sciencebase
-    remoteItemId: "5845ffaae4b04fc80e523664"
+    remoteItemId: "584aa8b4e4b07e29c706dd57"
```
Comment (ldecicco-USGS) [inline viz.yaml:122]: So, if you move your file over to the private sb repo, then change this back (I think?)

Comment (dblodgett-usgs) [reply viz.yaml:122]: We can just move this item into the viz community.

Comment (ldecicco-USGS) [reply viz.yaml:122]: whatever's easiest for you

Comment (dblodgett-usgs) [reply viz.yaml:122]: All good. It's in the community now.

**Pull request merged**